integration-tests/src/test/resources/domain13/custom/config.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<domain xmlns="http://xmlns.oracle.com/weblogic/domain" xmlns:f="http://xmlns.oracle.com/weblogic/domain-fragment" xmlns:s="http://xmlns.oracle.com/weblogic/situational-config">
+  <log>
+    <file-name f:combine-mode="add">/shared/logs/base_domain.log</file-name>
+  </log>
+  <server>
+    <name>admin-server</name>
+    <log f:combine-mode="add">
+      <file-name f:combine-mode="add">/shared/logs/admin-server.log</file-name>
+    </log>
+    <listen-address f:combine-mode="replace">domain13-admin-server</listen-address>
```

Comment:
It isn't acceptable for custom sit-cfg to override listen address or log files.  

See the custom sit config writeup in the site documentation.

Reply:
@tbarnes-us This config comes from the output of the introspector, I pretty much took the output in the job files folder.

Sit-Cfg-CFG--introspector-situational-config.xml
  <d:server>
    <d:name>admin-server</d:name>
    <d:log f:combine-mode="add">
      <d:file-name f:combine-mode="add">/shared/logs/admin-server.log</d:file-name>
    </d:log>
    <d:listen-address f:combine-mode="replace">domain1-admin-server</d:listen-address>
    <d:network-access-point>
      <d:name>T3Channel1</d:name>
      <d:listen-address f:combine-mode="replace">domain1-admin-server</d:listen-address>
    </d:network-access-point>
    <d:network-access-point>
      <d:name>T3Channel2</d:name>
      <d:listen-address f:combine-mode="replace">domain1-admin-server</d:listen-address>
    </d:network-access-point>
  </d:server>

Reply:
There are two kinds of sit-cfg overrides:
1) custom sit-cfg 
2) introspector generated
The former isn't needed in your testing.
The latter isn't needed, as the introspector is generating this automatically.

Reply:
The latter needs to be _properly_ generated, and subsequently consumed by the operator, and finally the operator should expose the port(s) via services.   It's the last that needs to be tested.

Reply:
I removed all of this